src/InertialSense.cpp

```diff
@@ -1380,239 +1379,6 @@ void InertialSense::CloseSerialPorts(bool drainBeforeClose)
 }
 
 #if 0
```

Comment:
Should these SaveFlashConfig be another wrapper into WithDevice() and call the ISDevice version?